frontend/src/app/(protected)/experiments/add/components/basicInfo.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+  Field,
+  FieldGroup,
+  Fieldset,
+  Label,
+  Description,
+} from "@/components/catalyst/fieldset";
+import { useState, useEffect, useCallback } from "react";
+import { Radio, RadioField, RadioGroup } from "@/components/catalyst/radio";
+import { Input } from "@/components/catalyst/input";
+import { Textarea } from "@/components/catalyst/textarea";
+import { AllSteps } from "./addExperimentSteps";
+import { useExperiment } from "./AddExperimentContext";
+import { Heading } from "@/components/catalyst/heading";
+import {
+  MABExperimentState,
+  NewMABArm,
+  NewABArm,
+  ABExperimentState,
+  StepValidation,
+} from "../../types";
+
+type Methods = typeof AllSteps;
+
+export default function AddBasicInfo({
+  setMethodType,
+  onValidate,
+}: {
+  setMethodType: (method: keyof Methods) => void;
+  onValidate: (validation: StepValidation) => void;
+}) {
+  const { experimentState, setExperimentState } = useExperiment();
+  const [errors, setErrors] = useState({
+    name: "",
+    description: "",
+    methodType: "",
+  });
+
+  const defaultMABArms: NewMABArm[] = [
+    { name: "", description: "", alpha_prior: 1, beta_prior: 1 },
+    { name: "", description: "", alpha_prior: 1, beta_prior: 1 },
+  ];
+  const defaultABArms: NewABArm[] = [
+    { name: "", description: "", mean_prior: 0, stdDev_prior: 1 },
+    { name: "", description: "", mean_prior: 0, stdDev_prior: 1 },
+  ];
+
+  const methodSelect = (value: keyof Methods) => {
+    setMethodType(value);
+    // TODO: It's not clean to have this component worr about each experiment type,
```

Comment:
I'll fix this in a future PR. I need to just figure out where this makes most sense